be/src/exec/exec_node.cpp

```diff
@@ -164,9 +164,7 @@ Status ExecNode::init(const TPlanNode& tnode, RuntimeState* state) {
         profile = print_plan_node_type(tnode.node_type);
     }
     init_runtime_profile(profile);
-
-    if (tnode.__isset.vconjunct) {
```

Comment:
I think this is not unused code. It reserved for vectorized engine(easy to merge code base)

Reply:
ok，i understand